internal/cmd/gocdk/apply.go

```diff
@@ -20,12 +20,14 @@ import (
 	"os"
 	"os/exec"
 	"path/filepath"
+	"strconv"
 
 	"golang.org/x/xerrors"
 )
 
 func apply(ctx context.Context, pctx *processContext, args []string) error {
 	f := newFlagSet(pctx, "apply")
+	input := f.Bool("input", true, "ask for input for Terraform variables if not directly set")
```

Comment:
`prompt`? `allow_input`?

Reply:
I'm matching Terraform's `-input` flag. I'd like to avoid inventing our own name for this flag.